src/components/features/pages/change-password/PasswordInfoSection/index.tsx

```
@@ -6,6 +6,10 @@ import Image from "next/image";
 import Cancel from "@/icons/Cancel.svg";
 import { usePasswordInfo } from "@/hooks/feature/info/usePasswordInfo";
 import Button from "@shared/ui/Button";
+import CancelIcon from "@icons/CancelIcon";
+import Spacing from "@shared/layout/Spacing";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Spacing'. Consider to delete it or rename it.

'Spacing' is defined here:


(lint/suspicious/noRedeclare)